sql/report_queries/transactions/transaction_report.sql

```diff
@@ -0,0 +1,173 @@
+        '' :: VARCHAR AS encumbrance_status, -- Enter the status of the encumbrance, e.g. Released, Unreleased, Pending
+        NULL :: BOOLEAN AS encumbrance_subscription,
+        --true :: BOOLEAN AS encumbrance_subscription, -- true or false
+        --
+        -- filters for finance data
+        '' :: VARCHAR AS budget_name, -- Enter your budget name
+        '' :: VARCHAR AS budget_status, -- Enter your budget status
+        '' :: VARCHAR AS fund_code, -- Enter your fund code
+        '' :: VARCHAR AS fund_name, -- Enter your fund name
+        '' :: VARCHAR AS fund_status, -- Enter your fund status
+        '' :: VARCHAR AS ledger_code, -- Enter your ledger code
+        '' :: VARCHAR AS ledger_name, -- Enter your ledger name
+        '' :: VARCHAR AS ledger_status, -- Enter your ledger status
+        '' :: VARCHAR AS fiscal_year_code, -- Enter your fiscal year code
+        '' :: VARCHAR AS fiscal_year_name, -- Enter your fiscal year name
+        --
+        -- filters on transaction level
+        '' :: VARCHAR AS transaction_source, -- Enter the source of the transaction, e.g. User, PoLine, Invoice
+        '' :: VARCHAR AS transaction_type -- Enter the type of the transaction, e.g. Credit, Encumbrance, Payment, Pending payment
+)
+SELECT 
+    finance_transactions.id AS transaction_id,
+    finance_transactions.metadata__created_date AS transaction_created_date,
+    finance_transactions.metadata__updated_date AS transaction_updated_date,
+    finance_transactions.source,  
+    finance_transactions.transaction_type,    
+    finance_expense_classes.code AS expense_class_code,
+    finance_expense_classes.name AS expense_class_name,
+    finance_transactions.currency,
+    finance_transactions.amount,
+    finance_transactions.encumbrance__amount_awaiting_payment,
+    finance_transactions.encumbrance__amount_expended,
+    finance_transactions.encumbrance__initial_amount_encumbered, 
+    finance_transactions.encumbrance__order_status,
+    finance_transactions.encumbrance__order_type,
+    COALESCE(finance_transactions.encumbrance__re_encumber, 'false') :: BOOLEAN AS encumbrance__re_encumber,  
+    finance_transactions.encumbrance__status,
+    COALESCE(finance_transactions.encumbrance__subscription, 'false') :: BOOLEAN AS encumbrance__subscription, 
+    finance_funds.fiscal_year_code,
+    finance_funds.fiscal_year_name,
+    finance_funds.fiscal_year_period_start,
+    finance_funds.fiscal_year_period_end,
+    finance_funds.budget_name,
+    finance_funds.budget_status,
+    finance_funds.fund_code,
+    finance_funds.fund_name,
+    finance_funds.fund_status,
+    finance_funds.fund_type_name,
+    finance_funds.ledger_code,
+    finance_funds.ledger_name,
+    finance_funds.ledger_status
+FROM 
+    public.finance_transactions 
+    LEFT JOIN public.finance_expense_classes ON finance_expense_classes.id = json_extract_path_text(finance_transactions.data, 'expenseClassId')
```

Comment:
Everything seems to work fine. Just one question:
The Checklists asks for a standardized format for extracting json data.
For LDP 1&2 it suggests: t #>> '{f1,f2,f3}'.
This report however uses jsonb_extract_path_text(t, f1, f2, f3) for both ldp and metadb.
I'm just wondering if it will function like that consistantly as well.


All queries:
- [x] PR Title and Description are accurate and thorough
- [x] PR is based on a new branch (not main)
- [x] PR scope is not overly broad
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary
- [ ] JSON extraction is in standard form, for example:
      LDP:     t #>> '{f1,f2,f3}'    [for compatibility across LDP 1 & 2]
      Metadb:  jsonb_extract_path_text(t, f1, f2, f3)

Report queries:
- [x] Query has complete user documentation
    - [x] Purpose of report
    - [x] Sample output
    - [x] Query instructions

Reply:
This is a good point.  I think we can be less strict about queries under report_queries/ since they will often have to be adjusted anyway in a specific library.

Reply:
Unfortunately, the queries were created before we had the new regulations and the new reporting app. If requested, I can update the queries according to the new specifications. At the time of the PR, there was no reporting app and the report queries were stored in subfolders. The question is whether the query for the new app could perhaps be completely recreated. 
This time we are so behind on the reviews that we have new regulations again. :-)

Reply:
Then I see no reason why I schouldn't approve if the queries are going to be adjusted anyway.
Or do you want to change it @stdombek ?

Reply:
We leave this query in this form. If the new variant for the reporting app is available, a separate query is created.